fv3gfs/wrapper/physics_properties.json

```diff
@@ -455,5 +462,26 @@
         "units": "unknown",
         "container": "Sfcprop",
         "dims": ["z_soil", "y", "x"]
+    },
+    {
+        "name": "total_sky_downward_longwave_flux_at_surface_override",
```

Comment:
Please rename to `override_for_total_(...)` to read more naturally, similarly for the other variables with `_override`.

Reply:
Sure thing -- done.